src/styles/index.scss

```diff
@@ -1,3 +1,127 @@
+$main-accent: #060B35;
+$secondary: #616070;
+$blue-accent: #00ACDC;
```

Comment:
Move variables to separate file

Reply:
isn't fixed. you should create a new `variables.scss` file, move this variables there and import this file to index.scss

Reply:
not fixed